_episodes/03-data-structures.md

```diff
@@ -11,18 +11,36 @@ objectives:
 - "To be familiar with differentiating data structures for choosing a good
    chart"
 keypoints:
-- "'Variables' are associated with columns in tidy data."
-- "As a descriptor of a specific column of data, 'variables' tend to describe
-  numerically continuous data, while 'categories' describe discrete or
+- "__Variables__ are associated with columns in tidy data."
+- "As a descriptor of a specific column of data, __variables__ tend to describe
+  numerically continuous data, while __categories__ describe discrete or
   categorical data that is organized in groups."
-- "'Values' are associated with cells in a data table."
-- "'Replicates' are values with similar variables and categories most often seen  
-  in experimental data."
-- "'Absolute' values receive context by their units, while 'relative' values are 
-  standardized in some fashion (i.e., proportion, per unit) for comparison among
-  categories."
+- "__Values__ are associated with cells in a data table."
+- "__Replicates__ are values with similar variables and categories most often
+  seen in experimental data."
+- "__Absolute__ values receive context by their units, while __relative__ values
+  are standardized in some fashion (i.e., proportion, per unit) for comparison
+  among categories."
 ---
 
+## Data structure is the shape and content of your data.
+
+The shape of your data ought to be rectangular and can be described by the number of rows and columns.
+
+Column in your data can also be called __variables__ or __fields__. However, the general use of the term __variable__ to describe a column can also be used more specifically to describe numerically continuous data. As a complement to this specific use of variables, __categories__ describe discrete or categorical data
```

Comment:
No change now, but you can also use `[variables]({{site.baseurl}}/reference/#variable)` to refer to a glossary entry in `reference.md`.